app/assess/templates/macros/comments_summary.html

```diff
@@ -0,0 +1,28 @@
+{% macro comment_summary(comments, themes) %}
+<div class="govuk-!-margin-top-7" id="anchor">
+    <h2 class="govuk-heading-m">Comments</h2>
+    <div id="more-detail-hint" class="govuk-hint">
+      <p class="govuk-hint">Summarise any thoughts you have on the information provided.</p>
+      <p class="govuk-hint">You cannot edit or delete your comment once you have saved it.</p>
+    </div>
+    {% if comments == None %}
```

Comment:
If we changed `comments` to a `dict`, I don't think this will ever work since it will at least be an empty `dict` - maybe a truthy of `if not comments` works better?

Reply:
comments now returns None due to the dictionary not actually being empty even if it had no comments